fdbclient/NativeAPI.actor.cpp

```diff
@@ -199,7 +199,21 @@ void DatabaseContext::addSSIdTagMapping(const UID& uid, const Tag& tag) {
 
 void DatabaseContext::getLatestCommitVersions(const Reference<LocationInfo>& locationInfo,
                                               Version readVersion,
+                                              const TransactionInfo& info,
                                               VersionVector& latestCommitVersions) {
+	latestCommitVersions.clear();
+
+	if (!info.readVersionObtainedFromGrvProxy) {
+		return;
+	}
+
+	if (readVersion > ssVersionVectorCache.getMaxVersion()) {
+		TraceEvent("GetLatestCommitVersions")
```

Comment:
Use `SevError` instead of assert?

```suggestion
		TraceEvent(SevError, "GetLatestCommitVersions")
```

Reply:
Will a simulation test still fail if we use SevError here? Maybe I will use ASSERT until we get a clean simulation test run.

Actually, some simulation tests did hit this assert and I am currently investigating the cause of that.

Reply:
The simulation will fail if `--crash` is used for any `SevError` events. I'm fine either way, to keep or to remove the assertion. Since you asserts false, a `SevError` is the appropriate thing to do.

Reply:
Some simulation tests are failing on this assertion. Will make this change as part of that patch.

Reply:
Done.

I am investigating those failures right now. They may go into a separate PR. Not sure at this point.